worker/process.ml

```diff
@@ -21,7 +21,7 @@ let pp_signal f x =
   else if x = sigterm then Fmt.string f "term"
   else Fmt.int f x
 
-let exec ~label ~log ~switch ?env ?(stdin="") ?(stderr=`FD_copy Unix.stdout) cmd =
+let exec ~label ~log ~switch ?env ?(stdin="") ?(stderr=`FD_copy Unix.stdout) ?(status=(function n -> n = 0)) cmd =
```

Comment:
I don't thing `status` is very clear. Maybe something like:
```suggestion
let exec ~label ~log ~switch ?env ?(stdin="") ?(stderr=`FD_copy Unix.stdout) ?(is_success=((=) 0)) cmd =
```

Reply:
Done.